### PR TITLE
K8SPG-66 PG db chart

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: charts/pg-db/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: pg-db
+description: 'A Helm chart for Deploying the Percona PostgreSQL database by Percona PostgreSQL operator'
```
Comment (fiowro) [inline charts/pg-db/Chart.yaml:3]:
```suggestion
description: 'A Helm chart for Deploying the Percona PostgreSQL database by Percona Distribution for PostgreSQL Operator'
```

File: charts/pg-db/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+# pg-db: A chart for installing Percona Distribution for PostgreSQL Operator managed Databases
+
+This chart implements Percona Distribution for PostgreSQL deployment in Kubernets via Custom Resource object. The project itself can be found here:
```
Comment (fiowro) [inline charts/pg-db/README.md:3]:
```suggestion
This chart implements Percona Distribution for PostgreSQL deployment on Kubernetes via a Custom Resource object. The project itself can be found here:
```

File: charts/pg-db/README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+* <https://github.com/percona/percona-postgresql-operator>
+
+## Pre-requisites
+
+* [PG operator](https://hub.helm.sh/charts/percona/pg-operator) running in you K8S cluster
```
Comment (fiowro) [inline charts/pg-db/README.md:9]:
```suggestion
* [Percona Distribution for PostgreSQL Operator](https://hub.helm.sh/charts/percona/pg-operator) running in you K8S cluster
```

File: charts/pg-db/README.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+| `backup.storages.storage0.bucket`                     | `Should be stated explicitly` Storage bucket |``|
+| `backup.storages.storage0.name`                     | `Should be stated explicitly for S3` S3-compatible storage name |``|
+| `backup.storages.storage0.type`                     | `Should be stated explicitly for S3` S3-compatible storage type |`s3`|
+| `backup.storages.storage0.endpointUrl`              | `Should be stated explicitly for S3` S3-compatible storage endpoint |``|
+| `backup.storages.storage0.region`                     | `Should be stated explicitly for S3` S3-compatible storage region |``|
+| `backup.storages.storage0.uriStyle`                     | (Optional) S3-compatible storage URI style |`path`|
+| `backup.storages.storage0.verifyTLS`                     | (Optional) S3-compatible storage URI style |`true`|
+| `pgBouncer.image`                     | Set this variable if you need to use custom pgbouncer image                                        | `percona/percona-postresql-operator:0.2.0-ppg13-pgbouncer`                              |
```
Comment (fiowro) [inline charts/pg-db/README.md:85]:
```suggestion
| `pgBouncer.image`                     | Set this variable if you need to use a custom pgbouncer image                                      | `percona/percona-postresql-operator:0.2.0-ppg13-pgbouncer`                              |
```

File: charts/pg-db/README.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+| `pgBouncer.size`                     | The number of pgbouncer instanses                                        | `1`                              |
+| `pgBouncer.resources.requests.cpu`                     | Container resource request limits CPU                                        | `1`                              |
+| `pgBouncer.resources.requests.memory`                     | Container resource request limits RAM                                        | `128Mi`                              |
+| `pgBouncer.resources.limits.cpu`                     | Container resource limits CPU                                        | `2`                              |
+| `pgBouncer.resources.limits.memory`                     | Container resource limits RAM                                        | `512Mi`                              |
+| `pgBouncer.expose.serviceType`                     | K8S service type for pgbouncer deployment | `ClusterIP`                              |
```
Comment (fiowro) [inline charts/pg-db/README.md:91]:
```suggestion
| `pgBouncer.expose.serviceType`                     | K8S service type for the pgbouncer deployment | `ClusterIP`                              |
```

File: charts/pg-db/README.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+| `secrets.postgres`                     | postges user password (superuser, not accessible via pgbouncer)                                       |`autogrenerated by operator`|
+| `secrets.pgbouncer`                     | pgbouncer user password                                        |`autogrenerated by operator`|
+| `secrets.<default_user>`                     | Default user password                                        |`autogrenerated by operator`|
+
+Specify parameters using `--set key=value[,key=value]` argument to `helm install`
+Notice that you can use multiple replica sets only with sharding enabled.
+
+## Examples
+
+### Deploy a replica set with disabled backups and no mongos pods
```
Comment (fiowro) [inline charts/pg-db/README.md:115]: mongos ?

File: charts/pg-db/README.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+NAME: dev
+LAST DEPLOYED:
+NAMESPACE: pgdb
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+```
+
+You can start up cluster with only S3 backup storage like this
```
Comment (fiowro) [inline charts/pg-db/README.md:130]:
```suggestion
You can start up the cluster with only S3 backup storage like this
```

File: charts/pg-db/README.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+```
+
+The chart can be customized using the following configurable parameters:
+
+| Parameter                       | Description                                                                   | Default                                   |
+| ------------------------------- | ------------------------------------------------------------------------------| ------------------------------------------|
+| `pause`                         | Stop PostgreSQL Database safely                                                    | `false`                                   |
+| `standby`                         | Switch/start PostgreSQL Database in standby mode                                                    | `false`                                   |
+| `disableAutofail`                         | Disables the high availability capabilities of a PostgreSQL cluster                                                    | `false`                                   |
+| `keepData`                         | Keep database container PVC intact after cluster removal                                                    | `false`                                   |
```
Comment (fiowro) [inline charts/pg-db/README.md:44]:
```suggestion
| `keepData`                         | Keep database container PVC intact after the cluster removal                                                | `false`                                   |
```

File: charts/pg-db/README.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+| `keepBackups`                         | Keep backups intact after cluster removal                                                    | `false`                                   |
```
Comment (fiowro) [inline charts/pg-db/README.md:45]:
```suggestion
| `keepBackups`                         | Keep backups intact after the cluster removal                                                | `false`                                   |
```

File: charts/pg-db/README.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+| `restoreFrom`                         | Set cluster/backup name as data source for newly created cluster                                                    | `false`                                   |
```
Comment (fiowro) [inline charts/pg-db/README.md:46]:
```suggestion
| `restoreFrom`                         | Set cluster/backup name as data source for a newly created cluster                                                  | `false`                                   |
```

File: charts/pg-db/README.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+| `restoreOpts`                         | Set restore (`pgBackrest`) options for restore action                                                    | `false`                                   |
+| `defaultUser`                         | Default unprivileged database username                                                    | `pguser`                                   |
+| `defaultDatabase`                         | Default user database created from cluster start                                                    | `pgdb`                                   |
```
Comment (fiowro) [inline charts/pg-db/README.md:49]:
```suggestion
| `defaultDatabase`                         | Default user database created from the cluster start                                                | `pgdb`                                   |
```

File: charts/pg-db/Chart.yaml (hunk 1 above)
Comment (cap1984) [reply charts/pg-db/Chart.yaml:3]: done

File: charts/pg-db/README.md (hunk 2 above)
Comment (cap1984) [reply charts/pg-db/README.md:3]: done

File: charts/pg-db/README.md (hunk 3 above)
Comment (cap1984) [reply charts/pg-db/README.md:9]: done

File: charts/pg-db/README.md (hunk 4 above)
Comment (cap1984) [reply charts/pg-db/README.md:85]: done

File: charts/pg-db/README.md (hunk 5 above)
Comment (cap1984) [reply charts/pg-db/README.md:91]: done

File: charts/pg-db/README.md (hunk 6 above)
Comment (cap1984) [reply charts/pg-db/README.md:115]: done

File: charts/pg-db/README.md (hunk 7 above)
Comment (cap1984) [reply charts/pg-db/README.md:130]: done

File: charts/pg-db/README.md (hunk 8 above)
Comment (cap1984) [reply charts/pg-db/README.md:44]: done

File: charts/pg-db/README.md (hunk 9 above)
Comment (cap1984) [reply charts/pg-db/README.md:45]: done

File: charts/pg-db/README.md (hunk 10 above)
Comment (cap1984) [reply charts/pg-db/README.md:46]: done

File: charts/pg-db/README.md (hunk 11 above)
Comment (cap1984) [reply charts/pg-db/README.md:49]: done

**Review by fiowro — APPROVED** [review]
lgtm

**Review by hors — APPROVED** [review]

**Pull request merged**